### ISSUE TITLE
Will the epoch setting affect the initial training loss?

### ISSUE THREAD

**Issue by gjxgjxgjxgjx** [open-mmlab/mmdetection#3411]
Why is the initial training loss different when changing the size of the epoch? For example, I set the epoch to 12. At the beginning, the loss was abnormal (negative number), but after changing the epoch to 24, the loss was normal again.

I am training FCOS. When it is set to 36epoch, it can be trained normally. I want to try to set it to 12, but the problem occurred at the beginning(Loss becomes negative).

**Comment by hellock** [comment]
That's abnormal. The total epochs should not affect the training for the first few epochs.

**Comment by gjxgjxgjxgjx** [comment]
@hellock Well, after trying many times, it may be a coincidence!
